### ISSUE TITLE
Internet Connection Sharing issue

### ISSUE THREAD

**Issue by kalagan-75** [henrypp/simplewall#691]
Hi,
I configured a VM on Hyper-V on my W10 notebook to use an "internal" network and I managed it to connect to internet using Internet Connection Sharing on wi-fi host network adapter.
Everything works perfectly when I disable filters on SimpleWall; if I enable it, the VM cannot connect anywhere, except IP addresses on 192.168.137.x subnet (ICS own subnet).

Any help would be appreciated: I'm not able to create a simplewall rule to allow the communication. 

PS: Simplewall is already configured to not disable windows firewall (required by ICS).
Thanks

**Comment by kalagan-75** [comment]
Just an update.
It seems that the VM can connect correctly if I close SimpleWall main application (leaving all filters on); if I start it, connections stop working. Quite strange.
![simplewall not running](https://user-images.githubusercontent.com/62688741/84391647-8e403380-abf9-11ea-8cc7-84617720ae20.png)
![simplewall running](https://user-images.githubusercontent.com/62688741/84391654-8f716080-abf9-11ea-83f0-10e8df74322c.png)

**Comment by kalagan-75** [comment]
Just another (and last) update: I give up on using ICS. It is too unstable.
Instead, I solved my problem using new-netnat powershell cmdlet for the internal virtual switch address translation.
This works perfectly (VM connected to internal network virtual switch and simplewall running with filters on can reach internet correctly)
Just for completeness (if others experience the same issue):
1. create a new internal virtual switch (this will create a new network connection named "vEthernet (virtual_switch_name)")
2. assign an IP address to the new network connection (in PS, New-NetIPAddress -IPAddress your_internal_ip  -PrefixLength 24  -InterfaceAlias 'vEthernet (virtual_switch_name)')
3. create the network nat translation (in PS, New-NetNat -Name your_nat_name -InternalIPInterfaceAddressPrefix your_internal_subnet/24)
4. assign your VM to use the new internal virtual switch
5. assign to your VM an IP address based to your_internal_subnet, and your_internal_ip as default gateway

Solved and closed
